.travis.yml

```diff
@@ -4,6 +4,7 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
+  - "3.7"
```

Comment:
Maybe just comment this out until the Travis issue is resolved?

Reply:
Just did that in 867dc24!